cookienotice/cookienotice.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * Name: Cookie Notice
+ * Description: Configure, show and handle a simple cookie notice
+ * Version: 1.0
+ * Author: Peter Liebetrau <https://socivitas/profile/peerteer>
+ * 
+ */
+use Friendica\Core\Addon;
+use Friendica\Core\Config;
+use Friendica\Core\L10n;
+
+function cookienotice_install()
+{
+	$file = 'addon/cookienotice/cookienotice.php';
+	Addon::registerHook('page_content_top', $file, 'cookienotice_page_content_top');
+	Addon::registerHook('page_end', $file, 'cookienotice_page_end');
+	Addon::registerHook('addon_settings', $file, 'cookienotice_addon_settings');
+	Addon::registerHook('addon_settings_post', $file, 'cookienotice_addon_settings_post');
+}
+
+function cookienotice_uninstall()
+{
+	$file = 'addon/cookienotice/cookienotice.php';
+	Addon::unregisterHook('page_content_top', $file, 'cookienotice_page_content_top');
+	Addon::unregisterHook('page_end', $file, 'cookienotice_page_end');
+	Addon::unregisterHook('addon_settings', $file, 'cookienotice_addon_settings');
+	Addon::unregisterHook('addon_settings_post', $file, 'cookienotice_addon_settings_post');
+}
+
+function cookienotice_addon_settings(&$a, &$s)
+{
+	if (!is_site_admin())
+		return;
+
+	/* Add our stylesheet to the page so we can make our settings look nice */
+
+	$a->page['htmlhead'] .= '<link rel="stylesheet"  type="text/css" href="/addon/cookienotice/cookienotice.css" media="all" />' . "\r\n";
+
+
+	$text = Config::get('cookienotice', 'text');
```

Comment:
You can pass a default value as 3rd parameter to the `Config::get` call.

Reply:
Ah, thats cool. Thank you.

Reply:
done